packages/site/src/content/InputFile/InputFileNotes.mdx

```diff
@@ -118,7 +126,194 @@ allowed to be uploaded.
 />
 ```
 
+## Basic usage
+
+```tsx
+<InputFile
+  getUploadParams={file => ({
+    url: "https://upload.example.com",
+    key: file.name,
+  })}
+/>
+```
+
+## Default content structure
+
+When no children are provided, InputFile renders different default content based
+on the `variation` and `size` props:
+
+### Dropzone variation (default)
+
+```tsx
+// size="base" (default)
+<InputFile getUploadParams={getUploadParams}>
+  <InputFile.DropzoneWrapper>
+    <Content spacing="small">
+      <InputFile.Button size="small" fullWidth={false} />
+      <InputFile.HintText />
+      <InputFile.Description />
+    </Content>
+  </InputFile.DropzoneWrapper>
+</InputFile>
+
+// size="small"
+<InputFile size="small" getUploadParams={getUploadParams}>
+  <InputFile.DropzoneWrapper>
+    <Content spacing="small">
+      <InputFile.Button size="small" fullWidth={false} />
+      {/* HintText and Description are omitted in small size */}
```

Comment:
```suggestion
      {/* In the default InputFile HintText and Description are omitted in small size */}
```

Makes it more clear they can be used in small variations but in the default implementations they aren't present. Also if you want to reword the suggestion I am fine with that